.npmignore

```diff
@@ -1,9 +1,3 @@
-favicon*
```

Comment:
not hosting a separate website for this... info should be on main codesherpas page